core-bundle/config/services.yaml

```diff
@@ -138,6 +138,11 @@ services:
     contao.csp.policy_manager:
         class: Nelmio\SecurityBundle\ContentSecurityPolicy\PolicyManager
 
+    contao.csp.wysiwyg_style_processor:
+        class: Contao\CoreBundle\Csp\WysiwygStyleProcessor
+        arguments:
+            - [text-decoration] # TODO: make this configurable via config.yaml
```

Comment:
```suggestion
            - {text-decoration: [underline, none]} # TODO: make this configurable via config.yaml
```
It’s very important to use an allowlist for the values too, as bad `url(…)` values is where CSS is most vulnerable. Allowing arbitrary values would also make our parser unsecure.

```suggestion
            - {text-decoration: 'underline|none', font-size: '[0-9]+(px|em)'} # TODO: make this configurable via config.yaml
```
Maybe as regex if needed? _(But I would prefer the non-regex version for security reasons)_

Reply:
I guess for properties like `font-size` we'd have to allow arbitrary values (regex).

Reply:
As tinyMCE only supports `['8pt', '10pt', '12pt', '14pt', '18pt', '24pt', '36pt']` the allowlist would still work here. But I guess it might still be too limited for other properties (I did not check all styles of our tinyMCE yet)

Reply:
Just checked. It has to be a regex because of the color properties.